reviews/2022-Piraeus-Datastore-annual.md

```diff
@@ -0,0 +1,128 @@
+as faster execution in comparison to userspace solutions, at the cost of some portability.
+
+## Highlights of Last Year
+###  Community
+
+Last year we saw the first major contributions from a community member:
+
+* Automatically setting up secured communications betweenc components with generated TLS certificates. [piraeus-operator#263](https://github.com/piraeusdatastore/piraeus-operator/pull/263)
+* Making our scheduler plugin independent of the STORK project. [github project](https://github.com/piraeusdatastore/linstor-scheduler-extender/)
+
+###  Releases
+
+Piraeus Datastore saw 4 main releases in the last year, with the occasional necessary patch release:
+
+* [v1.5.0](https://github.com/piraeusdatastore/piraeus-operator/releases/tag/v1.5.0) - May 12, 2021
+* [v1.6.0](https://github.com/piraeusdatastore/piraeus-operator/releases/tag/v1.6.0) - Sep 02, 2021
+* [v1.7.0](https://github.com/piraeusdatastore/piraeus-operator/releases/tag/v1.7.0) - Dec 14, 2021
+* [v1.8.0](https://github.com/piraeusdatastore/piraeus-operator/releases/tag/v1.8.0) - Mar 15, 2022
+
+## Annual Review Questions
+
+1. _Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats._
+
+    [Piraeus Datastore Devstats](https://piraeus.devstats.cncf.io/)
+
+    Highlights:
+    * [6 new contributors](https://piraeus.devstats.cncf.io/d/52/new-contributors-table?orgId=1&from=now-1y&to=now)
+    * [Commits](https://piraeus.devstats.cncf.io/d/2/commits-repository-groups?orgId=1&from=now-1y&to=now&var-period=d7&var-repogroups=All)
+     have remained steady in the last year.
+
+2. _How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)_
+
+    Piraeus Datastore has four [MAINTAINERS](https://github.com/piraeusdatastore/piraeus/blob/master/MAINTAINERS.md):
+
+   | Name | GithubID | Affiliation |
+   |------|----------|-------------|
+   | Alex Zheng | [alexzhc](https://github.com/alexzhc) | DaoCloud |
+   | Joel Colledge | [JoelColledge](https://github.com/JoelColledge) | LINBIT |
+   | Moritz Wanzenböck | [WanzenBug](https://github.com/WanzenBug) | LINBIT |
+   | Roland Kammerer | [rck](https://github.com/rck) | LINBIT |
+
+3. _What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are end users of your project, please do so. (Feel free to link to an existing ADOPTERS file if appropriate.)_
+
+    Piraeus Datastore does not directly track adoption. Some known adopters are listed here:
+   * [flant.com](https://flant.com/) in their [deckhouse platform](https://deckhouse.io/en/)
+   * [phoenix-media.eu](https://www.phoenix-media.eu)
```

Comment:
@alexzhc, I know you wanted to collect some more adopters. Just ping me if you have the information.

Reply:
@WanzenBug, I'd like to add another adoption case: [Fateflow Federated AI](https://github.com/rafflescity/rafflescity/blob/main/adoptions/federated_ai.md)